pytest-{{cookiecutter.plugin_name}}/.gitignore

```diff
@@ -59,6 +59,9 @@ instance/
 # Sphinx documentation
 docs/_build/
 
+# MkDocs documentation
+/site/
```

Comment:
I think this needs to be ``site/``, see https://github.com/mkdocs/mkdocs/blob/master/.gitignore#L1.